src/main/java/uk/gov/justice/digital/delius/controller/secure/CustodyResource.java

```diff
@@ -86,7 +86,7 @@ public IDs updateOffenderNomsNumber(final @PathVariable String crn,
 
     @RequestMapping(value = "offenders/nomsNumber/{originalNomsNumber}/nomsNumber", method = RequestMethod.PUT, consumes = "application/json")
     @ApiResponses(value = {
-            @ApiResponse(code = 400, message = "The new noms number is not present"),
+            @ApiResponse(code = 400, message = "The new noms number is not present in request or existing offender exists with then new NOMS number already"),
```

Comment:
could possibly model as a `409 Conflict`?

Reply:
@andrewrlee I like that a lot. @mikehalmamoj what do you think?

Reply:
:+1: